lib/plugins/aws/provider/awsProvider.js

```diff
@@ -341,6 +452,18 @@ class AwsProvider {
               required: ['localMountPath', 'arn'],
             },
             handler: { type: 'string' },
+            layers: { $ref: '#/definitions/awsLambdaLayers' },
+            memorySize: { $ref: '#/definitions/awsLambdaMemorySize' },
+            onError: { $ref: '#/definitions/awsSnsArn' },
+            provisionedConcurrency: { type: 'integer', minimum: 0 },
```

Comment:
In code we ignore `0` value, so I think it'll be more accurate to put `minimum: 1`